styleguide/source/_patterns/01-atoms/display-switch.twig

```diff
@@ -0,0 +1,3 @@
+<li class="ama__display-switch">
```

Comment:
This probably should be either a span or div so we can use it outside of a list

Reply:
Addressed in @JaeBea's comment; I've hidden the atom and kept it as an `<li>`. If it needs to be used outside a list, we'll need to address that later